src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -182,9 +192,13 @@ class JesBackend extends Backend with LazyLogging {
     val jesParameters = standardParameters(callGcsPath) ++ jesInputs ++ unsafeJesOutputs
 
     // Not possible to currently get stdout/stderr so redirect everything and hope the WDL isn't doing that too
-    val redirectedCommand = s"$instantiatedCommandLine > $LocalStdoutValue 2> $LocalStderrValue"
+    val redirectedCommand = s"$instantiatedCommandLine"
+
+    val bais = new ByteArrayInputStream(redirectedCommand.getBytes)
```

Comment:
I'm wondering if it's even worth doing this or just calling it in the uploadObject call. I think it'd depend on how the latter actually looked